apiserver/authwrapper/authorizer.go

```diff
@@ -12,31 +12,34 @@ import (
 	"k8s.io/apiserver/pkg/endpoints/filters"
 )
 
-//go:generate go run github.com/golang/mock/mockgen -destination=./mock/$GOFILE -package mock_organization k8s.io/apiserver/pkg/authorization/authorizer Authorizer
+//go:generate go run github.com/golang/mock/mockgen -destination=./mock/$GOFILE -package mock k8s.io/apiserver/pkg/authorization/authorizer Authorizer
 
-// rbacAuthorizer processes authorization requests for `organizations.organization.appuio.io` and checks them based on rbac rules for `organizations.rbac.appuio.io`
-type rbacAuthorizer struct {
-	Authorizer authorizer.Authorizer
+// Authorizer processes authorization requests for `{rbacNamespace.Resource}` and checks them based on rbac rules for `{rbacNamespace}`
+type Authorizer struct {
+	Authorizer    authorizer.Authorizer
+	rbacNamespace metav1.GroupVersionResource
```

Comment:
Nitpick: Should this really be called `rbacNamespace`. It doesn't really have anything to do with namespaces does it?

Reply:
> GroupVersionResource unambiguously identifies a resource

So i think the proper term would be `rbacID`?